src/Microsoft.DotNet.ProjectModel/ProjectContextBuilder.cs

```diff
@@ -37,6 +37,8 @@ public class ProjectContextBuilder
 
         private Func<string, LockFile> LockFileResolver { get; set; }
 
+        private ProjectReader.Settings Settings { get; set; }
```

Comment:
This isn't bad except for the fact that it's called ProjectReader.Settings and is exposed on the builder as that nested type.